src/Syntax.elm

```diff
@@ -62,7 +56,8 @@ type SyntaxType
     | UseKeyword
     | UsePrefix
     | UseSuffix
-    | HashQualifier HashQualified
+      -- TODO: Should this be NameOnly Name | HashOnly Hash | HashQualified Name Hash
```

Comment:
This is more of an open question for later—I'm not actually using the HashQualifier value for anything at the moment.

Reply:
You could try adding an ambiguous name to your codebase and observe how that comes out. It should use the `HashQualifier` structure.